source/adapters/cuda/context.cpp

```diff
@@ -98,13 +98,12 @@ UR_APIEXPORT ur_result_t UR_APICALL urContextGetInfo(
                            UR_MEMORY_SCOPE_CAPABILITY_FLAG_DEVICE;
     return ReturnValue(Capabilities);
   }
+  case UR_CONTEXT_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES:
+  case UR_CONTEXT_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES:
   case UR_CONTEXT_INFO_USM_MEMCPY2D_SUPPORT:
```

Comment:
looks like they were missed when this was first written so just making sure they are included to ensure they reteurn `UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION` instead of `UR_RESULT_ERROR_INVALID_ENUMERATION`

Reply:
I may be misreading the diff, but it looks like `UR_CONTEXT_INFO_USM_MEMCPY2D_SUPPORT` is being changed from `true` and `UR_CONTEXT_INFO_USM_FILL2D_SUPPORT` is being changed from `false`.

Reply:
oh yep you're right, my bad! `UR_CONTEXT_INFO_USM_MEMCPY2D_SUPPORT` shouldn't have been changed

for `UR_CONTEXT_INFO_USM_FILL2D_SUPPORT` I'm not sure what would be more correct to return `false` or `UR_RESULT_ERROR_UNSUPPORTED_ENUMERATION`

Reply:
We discussed this directly and confirmed simple boolean queries should use `return ReturnValue(false/true)`. I will update this soon.

Reply:
Is this done? If so resolve the conversation.